src/main/java/org/isf/security/jwt/TokenProvider.java

```diff
@@ -71,15 +73,21 @@ public class TokenProvider implements Serializable {
 
 	private JwtParser jwtParser;
 
+	@Autowired
+	private UserDetailsServiceImpl userDetailsService;
+
 	@PostConstruct
 	public void init() {
 		String secret = env.getProperty("jwt.token.secret");
 		LOGGER.info("Initializing JWT key with secret: {}", secret);
 		byte[] keyBytes = secret.getBytes(StandardCharsets.UTF_8);
 		this.key = Keys.hmacShaKeyFor(keyBytes);
 
-		this.tokenValidityInMilliseconds = 1000L * 6000;
-		this.tokenValidityInMillisecondsForRememberMe = 1000L * 6000;
+		// 15 minutes (900,000 milliseconds)
+		this.tokenValidityInMilliseconds = 1000L * 60 * 15;
+
+		// 7 days (604,800,000 milliseconds)
+		this.tokenValidityInMillisecondsForRememberMe = 1000L * 60 * 60 * 24 * 7;
```

Comment:
Toooo long! 3 days or less is enough I think.

Reply:
Thanks! I will make them configurable anyway